src/Avalonia.Controls/Application.cs

```diff
@@ -335,6 +337,11 @@ public virtual void RegisterServices()
                 .Bind<IScheduler>().ToConstant(AvaloniaScheduler.Instance)
                 .Bind<IDragDropDevice>().ToConstant(DragDropDevice.Instance)
                 .Bind<IPlatformDragSource>().ToTransient<InProcessDragSource>();
+
+            var clock = new RenderLoopClock();
+            AvaloniaLocator.CurrentMutable
+                .Bind<IClock>().ToConstant(clock)
```

Comment:
Feels a little strange binding `IClock` to the global clock instance as there can be many `IClock`s in the system. Maybe an `IGlobalClock` interface might be a good idea for clarity?

Reply:
I can add an empty `IGlobalClock` interface. My thought was that any clock instance could be a global clock, even a regular `Clock` as long as something is providing input to it. Unit tests would use a different clock for example.

Reply:
Yeah, but there's only one global clock at runtime right? Even if it might be different in unit tests, that's normal.

Reply:
I'll add the interface in.